app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -1533,6 +1605,11 @@ class PluginApi {
    * addPostClassesCallback((attrs) => {if (attrs.post_number == 1) return ["first"];})
    **/
   addPostClassesCallback(callback) {
+    deprecated(
+      "`api.addPostClassesCallback` has been deprecated. Use the value transformer `post-class` instead.",
+      POST_STREAM_DEPRECATION_OPTIONS
+    );
```

Comment:
Another to handle automatically? (admittedly `attrs` will need to be replaced by the post model, but that will be drop-in for most cases)

Reply:
Another one handled automatically

Reply:
Another one handled automatically